apps/platforms/win/windows_terminal/windows_terminal.talon

```diff
@@ -18,3 +18,13 @@ kill all:
     insert("y")
     key(enter)
     
+
+(find it | buffy | queer eye)$:
+    key(ctrl-shift-f)
```

Comment:
Could just call `edit.find()` once you handle the `None` case in its implementation.